tests/SkiaSharp.Wasm.Tests/Program.cs

```diff
@@ -1,11 +1,60 @@
-﻿namespace SkiaSharp.Tests
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using Xunit;
+
+namespace SkiaSharp.Tests
 {
 	public class Program
 	{
-		public static int Main()
+		private static readonly Assembly assembly = typeof(Program).Assembly;
+
+		public static int Main(string[] args)
 		{
+			Console.WriteLine($"args: {string.Join(" ", args)}");
+
+			// copy resources out of assembly
+			var prefix = "SkiaSharp.Tests.";
+			var resources = assembly.GetManifestResourceNames();
+			foreach (var resource in resources)
+			{
+				if (!resource.StartsWith(prefix))
+					continue;
+
+				if (resource.StartsWith(prefix + "fonts.") || resource.StartsWith(prefix + "images."))
```

Comment:
I might be wrong, but I think assemblies are in CamelCase notation, so this should be 
```suggestion
				if (resource.StartsWith(prefix + "Fonts.") || resource.StartsWith(prefix + "Images."))
```